### ISSUE TITLE
While using allure-commandline via npm, an error getting displayed

### ISSUE THREAD

**Issue by anarwal** [allure-framework/allure-core#749]
When I run npm install for allure-commandline it works fine, but then when I run allure from my commandline of IDE, it throws me following error:

```
C:\Users\anarwal\IdeaProjects\cm-ui-automation>allure help
events.js:154
throw er; // Unhandled 'error' event
^

Error: spawn C:\Users\anarwal\AppData\Roaming\npm\node_modules\allure-commandline\dist\bin\allure
ENOENT
at exports._errnoException (util.js:856:11)
at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
at onErrorNT (internal/child_process.js:344:16)
at nextTickCallbackWith2Args (node.js:474:9)
at process._tickCallback (node.js:388:17)
at Function.Module.runMain (module.js:449:11)
at startup (node.js:139:18)
at node.js:999:3
```
I have tried to look into the issue but found nothing so far, what can be wrong in here?

**Comment by just-boris** [comment]
It seems like a windows issue, because it works fine for me.

I will try to reproduce it again when I get a windows

**Comment by just-boris** [comment]
@anarwal also you can help, if you provide content of the `C:\Users\anarwal\AppData\Roaming\npm\node_modules\allure-commandline\dist\bin\` folder.

**Comment by anarwal** [comment]
@just-boris in that folder I have a file "allure" (I do not have a bat file of allure which I was able to see in bin folder for allure-commandline unzipped folder)

**Comment by anarwal** [comment]
@just-boris I believe there should be a allure.bat(required for windows) in my bin folder but when I do npm install for allure-commandline the bin folder created doesnt have that file

**Comment by just-boris** [comment]
@anarwal no, it shouldn't. NodeJS on windows creates a `.cmd` wrapper for each properly described script.
I will try to figure out why it didn't work.

**Comment by anarwal** [comment]
@just-boris thanks. let me know, as now its a pain for me to run reports externally from command prompt.

**Comment by just-boris** [comment]
Fixed in 1.4.22-1 (it is release on npm, not the allure-core itself)

@anarwal you can it using following
```
npm install allure-commandline@1.4.22-1
```

**Comment by anarwal** [comment]
It worked! Thanks
